### ISSUE TITLE
ImportError: No module named resources.wordlists

### ISSUE THREAD

**Issue by MeanMachine123** [threat9/routersploit#402]
:~/routersploit# python rsf.py 
Traceback (most recent call last):
  File "rsf.py", line 7, in <module>
    from routersploit.interpreter import RoutersploitInterpreter
  File "/root/routersploit/routersploit/interpreter.py", line 12, in <module>
    from routersploit.core.exploit.exceptions import RoutersploitException
  File "/root/routersploit/routersploit/core/exploit/__init__.py", line 1, in <module>
    from routersploit.core.exploit.exploit import (
  File "/root/routersploit/routersploit/core/exploit/exploit.py", line 14, in <module>
    from routersploit.core.exploit.option import Option
  File "/root/routersploit/routersploit/core/exploit/option.py", line 5, in <module>
    from routersploit.core.exploit.utils import (
  File "/root/routersploit/routersploit/core/exploit/utils.py", line 9, in <module>
    import routersploit.resources.wordlists as wordlists
ImportError: No module named resources.wordlists

**Comment by lucyoa** [comment]
We won't be able to figure out whats the problem if you cannot provide us with all required information. Create new issue with new issue template and fill all sections.

**Comment by Dom-1** [comment]
Okay, I'm having the same issue on Mac. If you see the layout of the directories, there shouldn't be a problem with locating the file, however, there is. Is there any possible way to come to a clear resolution?

**Comment by lucyoa** [comment]
You clearly cannot follow simple instructions on README page and u are using python2 instead of python3.

**Comment by Dom-1** [comment]
Yes. And I understand, I'm just simply trying to solve a problem that has been active for over 18 days now.
